File: app/src/main/java/com/example/pokeapitarea/PokeNet.java
package com.example.pokeapitarea;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface PokeNet {
    @GET("pokemon")
    default Call<Respond> obtenerListaPokemon(@Query("limit") int limit, @Query("offset") int offset) {
        return null;
    }
}

File: app/src/main/java/com/example/pokeapitarea/PokeNet.kt
package com.example.pokeapitarea

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface PokeNet {
    @GET("pokemon")
    fun obtenerListaPokemon(@Query("limit") limit: Int, @Query("offset") offset: Int): Call<Respond>? {
        return null
    }
}
